ansible/playbooks/roles/prometheus/defaults/main.yml

```diff
@@ -3,11 +3,13 @@ prometheus_external_labels:
   environment: "{{ ansible_fqdn }}"
 
 prometheus:
+  version: 2.31.1
   file_name:
-    x86_64: "prometheus-2.10.0.linux-amd64.tar.gz"
-    aarch64: "prometheus-2.10.0.linux-arm64.tar.gz"
+    x86_64: "prometheus-2.31.1.linux-amd64.tar.gz"
+    aarch64: "prometheus-2.31.1.linux-arm64.tar.gz"
 
 alertmanager:
+  version: 0.23.0
   file_name:
-    x86_64: "alertmanager-0.17.0.linux-amd64.tar.gz"
-    aarch64: "alertmanager-0.17.0.linux-arm64.tar.gz"
+    x86_64: "alertmanager-0.23.0.linux-amd64.tar.gz"
+    aarch64: "alertmanager-0.23.0.linux-arm64.tar.gz"
```

Comment:
In #2767 we have versions.yml, are we going to introduce same here? Not so important for me though.

Reply:
I don't think its needed for now, but we should set some standards when we move all this stuff to the collection. Standards like:

- Common bin/config directories for everything
- Standards for data directories
- version defaults as you mentioned above
- and probably a whole bunch more ...